marketdata/stream/client.go

```diff
@@ -33,6 +33,9 @@ type client struct {
 	in                 chan []byte
 	subChanges         chan []byte
 
+	lastBufferFillMsg time.Time
+	droppedMsgs       int
```

Comment:
Please do add a unit test for this change. `client_test.go` has some similar tests.

I'd prefer `lastBufferFillMsg` -> `lastBufferFill`, because it's not a timeout, it's the last time it was filled. I slightly prefer `droppedMsgs` over `droppedMsgsCount` or `droppedMsgCount` but either is fine.